lib/cictl/index_command.rb

```diff
@@ -42,27 +42,20 @@ def all
         puts "5 second delay if you need it..."
         sleep 5
       end
+
       # Make sure there is a full MARC file to work on
       preflight(last_full_marc_file)
+
       logger.info "Empty full Solr records"
       solr_client.empty_records!
-      logger.info "Load most recent set of deleted records into solr"
-      if DeletedRecords.most_recent_non_empty_file
-        logger.info "Found #{DeletedRecords.most_recent_non_empty_file}"
-        solr_client.send_jsonl(DeletedRecords.most_recent_non_empty_file)
-      else
-        logger.error "Can't find any non_empty deleted_record files in #{DeletedRecords.save_directory}"
-      end
-      logger.info "Commit"
-      solr_client.commit!
-      logger.info "Using full marcfile #{last_full_marc_file}"
-      # Calling the Thor "file" command.
-      call_file_command last_full_marc_file
+
+      load_deleted_records
+
+      index_full_file(last_full_marc_file)
+
       # "since" command for a month starts on the last day of last month
       # because there will generally be both an "upd" and a "full" file.
       call_since_command last_full_marc_file.to_datetime
-      logger.info "Commit"
```

Comment:
The individual date commands commit, so we don't need to commit again afterwards (although it doesn't hurt)